app/src/main/java/com/ishikota/photoviewerandroid/ui/top/TopFragment.kt

```diff
@@ -16,6 +20,23 @@ import com.ishikota.photoviewerandroid.infra.attachTabLayoutAdapter
 import com.ishikota.photoviewerandroid.ui.top.collectionlist.TopCollectionListFragment
 import com.ishikota.photoviewerandroid.ui.top.photolist.TopPhotoListFragment
 
+class BottomNavigationBehavior<V : View>(context: Context, attrs: AttributeSet) :
```

Comment:
ref https://android.jlelse.eu/scroll-your-bottom-navigation-view-away-with-10-lines-of-code-346f1ed40e9e